test/controllers/albums_controller_test.rb

```diff
@@ -0,0 +1,51 @@
+require 'test_helper'
+
+class AlbumsControllerTest < ActionController::TestCase
+  test "should get index" do
+    get :index
+    assert_response :success
+    assert_template :index
+  end
+
+  test "should get show" do
+    get :show, {id: albums(:one)}
+    assert_response :success
+  end
+
+  test "should get new" do
+    get :new
+    assert_response :success
+  end
+
+  test "should create an object" do
+    params = {album: {name: "michaelJ"}}
```

Comment:
All of the create and update tests for this controller will need to include an `artist` key along with the `name` key before they will pass. This is because we are not providing sufficient data for all of the validations to pass, so the controller cannot save the model.